src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx

```diff
@@ -71,6 +72,12 @@ const BrowserTrackConfig: FunctionComponent<BrowserTrackConfigProps> = (
     } else {
       props.updateTrackConfigNames(selectedCog, !trackOurConfigName);
     }
+
+    analyticsTracking.trackEvent({
+      category: 'track_settings',
+      label: selectedCog,
+      action: 'track_name_' + (trackOurConfigName ? 'on' : 'off')
```

Comment:
re.: `action: 'track_name_' + (trackOurConfigName ? 'on' : 'off')` — isn't it the opposite? Isn't the code saying "if there is a trackOurConfigName or if it is false, then turn it to `true`"? Therefore, shouldn't the action say "on"?

(Also, what does `trackOurConfigName` mean? Perhaps rename it to `shouldShowTrackName` (and `shouldShowTrackLabels` for `trackOurConfigLabel `)?

Reply:
Updated it. Thanks